src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -199,10 +199,17 @@ private void UnSubscribeWorkspaceEvents()
 
         private void UnsubscribeEvents()
         {
-            LoadedParams.CurrentWorkspaceChanged -= OnCurrentWorkspaceChanged;
-            DynamoViewModel.CurrentSpaceViewModel.Model.NodeAdded -= OnNodeAdded;
-            DynamoViewModel.Model.Logger.NotificationLogged -= OnNotificationLogged;
-            CurrentWorkspace.RequestPackageDependencies -= PythonDependencies.AddPythonPackageDependency;
+            if (LoadedParams != null)
```

Comment:
Were there null reference exceptions being thrown with all these objects earlier?

Reply:
yes during shutdown directly from the splash screen being closed.